include/os/freebsd/spl/sys/cmn_err.h

```diff
@@ -73,6 +74,30 @@ extern void vuprintf(const char *, __va_list)
 extern void panic(const char *, ...)
     __attribute__((format(printf, 1, 2), __noreturn__));
 
```

Comment:
I was just wondering if you should also add `vzcmn_err_once()` just for completeness? You seemed to cover all the other variants with a `once` version.

Reply:
Just a FYI in case people were not aware, `__VA_ARGS__` could be used to implement this. It is part of the C standard. GCC has fairly good documentation regarding it:

https://gcc.gnu.org/onlinedocs/cpp/Variadic-Macros.html

Reply:
Sure, actually I did add `vzcmn_err _once()` but missed the `v` and missed to add `zcmn_err()`. Thanks for catching that.

Reply:
@ryao No worries.